InfinityCosmos/ForMathlib/CategoryTheory/Enriched/Ordinary.lean

```diff
@@ -0,0 +1,30 @@
+import Mathlib.CategoryTheory.Enriched.Ordinary
+
+universe v' v u u'
+
+open CategoryTheory Category MonoidalCategory
+
+namespace CategoryTheory
+
+variable (V : Type u') [Category.{v'} V] [MonoidalCategory V]
+  {C : Type u} [Category.{v} C] [EnrichedOrdinaryCategory V C]
+
+def eHomWhiskerLeftIso (X : C) {Y Y' : C} (i : Y ≅ Y') :
+    (X ⟶[V] Y) ≅ (X ⟶[V] Y') where
+  hom := eHomWhiskerLeft V X i.hom
+  inv := eHomWhiskerLeft V X i.inv
+  hom_inv_id := by
+    rw [← eHomWhiskerLeft_comp, i.hom_inv_id, eHomWhiskerLeft_id]
+  inv_hom_id := by
+    rw [← eHomWhiskerLeft_comp, i.inv_hom_id, eHomWhiskerLeft_id]
```

Comment:
The (two-sided) unenriched analogue appears to be [`CategoryTheory.Iso.homCongr`](https://github.com/leanprover-community/mathlib4/blob/65c00859d05eb8ff717bb0f06ff355609be271b2/Mathlib/CategoryTheory/HomCongr.lean#L36). Perhaps this deserves a more complete treatment than I've given here (or at least a better name)?

Reply:
I added `eHomCongr` and `sHomCongr` to be more consistent with the unenriched version of the api here, but I stopped short of adding analogues for all of the additional lemmas in `Mathlib/CategoryTheory/HomCongr.lean`.

Reply:
Update: I implemented a more direct proof of `IsSLimit.ofIsoSLimit` that removed the need for `eHomCongr`. I think it's probably still a good thing to eventually have in mathlib, but it doesn't necessarily need to be in this PR.